docs/diagrams/DeleteSequenceDiagram.puml

```diff
@@ -5,12 +5,13 @@ box Logic LOGIC_COLOR_T1
 participant ":LogicManager" as LogicManager LOGIC_COLOR
 participant ":OneShelfBookParser" as AddressBookParser LOGIC_COLOR
 participant ":ItemDeleteCommandParser" as DeleteCommandParser LOGIC_COLOR
-participant "d:ItemDeleteCommand" as DeleteCommand LOGIC_COLOR
+participant ":ItemDeleteCommand" as DeleteCommand LOGIC_COLOR
 participant ":CommandResult" as CommandResult LOGIC_COLOR
 end box
 
-box Model MODEL_COLOR_T1
-participant ":Model" as Model MODEL_COLOR
+box Models MODEL_COLOR_T1
+participant ":InventoryModel" as Model MODEL_COLOR
+participant ":ModelManager" as ModelManager MODEL_COLOR
```

Comment:
There isn't a "ModelManager" class from what I remember. It should be "DeliveryModel".

Reply:
fixed in latest commit
https://github.com/AY2021S1-CS2103T-T12-1/tp/pull/169/commits/c276c055ccc67713af31fe5506ca399aa0cee012